python/paddle/distributed/fleet/base/topology.py

```diff
@@ -83,27 +95,27 @@ def __init__(
             zip(self._coord2rank.values(), self._coord2rank.keys())
         )
 
-    def get_hybrid_group_names(self):
+    def get_hybrid_group_names(self) -> list[str]:
         return self._parallel_names
 
-    def get_dim(self, axis_name):
+    def get_dim(self, axis_name: str) -> list[str]:
         return self._dims[self._parallel_names.index(axis_name)]
 
-    def world_size(self):
+    def world_size(self) -> int:
         return self._world_size
 
-    def get_rank(self, **args):
+    def get_rank(self, **args: Any) -> dict[list[str], int]:
         assert len(args) == len(self._dims)
         key = self.coordinate(**args)
         assert key in self._coord2rank.keys()
         return self._coord2rank[key]
 
-    def get_coord(self, rank):
+    def get_coord(self, rank: str) -> int | list[str] | None:
```

Comment:
```suggestion
    def get_coord(self, rank: int) -> Any:
```

返回的是个动态的 namedtuple ，用 `Any` 吧 ～